tools/please_pex/custom.py

```diff
@@ -0,0 +1,8 @@
+import importlib
+
+
+def run_tests(args):
+    """Runs tests using a custom test runner that is selected by the user."""
+    runner, _, name = "__TEST_RUNNER__".rpartition('.')
+    mod = importlib.import_module(runner)
+    return getattr(mod, name)(TEST_NAMES, args)
```

Comment:
Can we check the result of `getattr` is a function? It's likely to allow us to return a nicer error message than triggering a traceback with "TypeError: 'foo' object is not callable"